schematic/schemas/data_model_jsonld.py

```diff
@@ -0,0 +1,284 @@
+from functools import wraps
+from typing import Any, Dict, Optional, Text, List
+import networkx as nx
+
+from schematic.schemas.data_model_graph import DataModelGraphExplorer
+from schematic.schemas.data_model_relationships import DataModelRelationships
+from schematic.utils.schema_utils import get_label_from_display_name, convert_bool_to_str
+
+
+class DataModelJsonLD(object):
```

Comment:
I think the template object that gets referred to by several methods in this class should be its own class.

Reply:
Can you elaborate a bit more?

Reply:
Okay, I think the move to data classes might have addressed this, but let me know if you were thinking of something different.